packages/appkit/src/networks/ton/ton.ts

```diff
@@ -0,0 +1,15 @@
+import { defineChain } from '../utils.js'
+
+export const ton = defineChain({
+  id: '-239',
+  name: 'TON',
+  network: 'ton-mainnet',
+  nativeCurrency: { name: 'TON', symbol: 'TON', decimals: 9 },
+  rpcUrls: {
+    default: { http: ['https://rpc.walletconnect.org/v1'] }
```

Comment:
should we keep tonkeeper here? so it's persisted as a backup? we would be setting bapi anyways when extending the network right?